dashboard/db/migrate/20160706041025_add_started_to_plc_course_unit.rb

```diff
@@ -0,0 +1,7 @@
+class AddStartedToPlcCourseUnit < ActiveRecord::Migration
+  def change
+    add_column :plc_course_units, :started, :boolean
+
+    Plc::CourseUnit.update_all(started: true)
```

Comment:
Yes, as of right now. Although now that I think about it, that might not be the case when this gets checked in. I'm fine removing it and doing this manually